nearcore/src/runtime/mod.rs

```diff
@@ -3199,4 +3201,27 @@ mod test {
         let view_state_value = view_state.get(&key).unwrap().unwrap();
         assert_eq!(state_value, view_state_value);
     }
+
+    /// Check that mainnet genesis hash still matches.
+    #[test]
+    fn test_genesis_hash() {
+        let genesis = Genesis::from_file("res/mainnet_genesis.json", GenesisValidationMode::Full);
+        let chain_genesis = ChainGenesis::new(&genesis);
+        let (tempdir, opener) = near_store::NodeStorage::test_opener();
+        let store = opener.open().unwrap();
+
+        let runtime = Arc::new(NightshadeRuntime::test_with_runtime_config_store(
+            tempdir.path(),
+            store.get_store(Temperature::Hot),
+            &genesis,
+            TrackedConfig::new_empty(),
+            RuntimeConfigStore::new(None),
+        ));
+
+        let block = Chain::make_genesis_block(&*runtime, &chain_genesis).unwrap();
+        assert_eq!(
+            block.header().hash(),
+            &CryptoHash::from_str("EPnLgE7iEq9s7yTkos96M3cWymH5avBAPm3qx3NXqR8H").unwrap()
```

Comment:
```suggestion
            &"EPnLgE7iEq9s7yTkos96M3cWymH5avBAPm3qx3NXqR8H".parse::<CryptoHash>().unwrap()
```


more idiomatic form, and avoids the import (which I think will trip unused warning in the current code on CI)

Reply:
Alternatively, `block.header().hash().to_string()` and compare with encoded hash.

Reply:
done.